### DISCUSSION TITLE
💡Root Installation Method

### DISCUSSION THREAD

**Discussion by qomarhsn** [NeoApplications/Neo-Store#554] [Ideas]
Hey there, Neo Store developers! 👋 I've been using Neo Store for a few days now and I have to say, it's been an amazing experience! 🌟 I wanted to share an idea with you that I believe could make the app even better and more user-friendly. Currently, when we install any software from Neo Store, we only have the option for the normal install method. This means we have to manually click the install button twice during the installation process. It would be fantastic if you could consider adding a root installation method to Neo Store. Here's how it could work: for users who have rooted their phones and are comfortable granting root access, Neo Store could utilize this access to install apps seamlessly in the background. This would eliminate the need for users to go through the manual installation process and enhance the overall convenience of using Neo Store. I believe that incorporating a root installation method would be a game-changer for those who have rooted their phones and would make Neo Store even more appealing to them. Thank you for considering this suggestion! I truly appreciate the effort you put into making Neo Store an amazing app, and I'm excited to see how this idea could potentially improve the user experience. Keep up the great work! 🚀💪 📱💡✨

**Comment by machiav3lli** [comment]
NS already supports root installer, check the downloads/installer settings

**Reply by qomarhsn** [reply]
thanks!
